src/common/wrap/wrap-events.js

```diff
@@ -45,7 +45,8 @@ export function wrapEvents (sharedEE) {
   ee.on(ADD_EVENT_LISTENER + '-start', function (args, target) {
     var originalListener = args[1]
     if (originalListener === null ||
-      (typeof originalListener !== 'function' && typeof originalListener !== 'object')
+      (typeof originalListener !== 'function' && typeof originalListener !== 'object') ||
+      args[0] === 'mousemove'
```

Comment:
I think wrap-events doesn't need changing. It just needs to be excluded from Traces. There's no event specific logic in here